sound/soc/sof/core.c

```diff
@@ -365,6 +365,15 @@ int snd_sof_device_probe(struct device *dev, struct snd_sof_pdata *plat_data)
 	if (sof_core_debug)
 		dev_info(dev, "sof_debug value: %#x\n", sof_core_debug);
 
+	if (sof_debug_check_flag(SOF_DBG_DSPLESS_MODE)) {
+		if (plat_data->desc->dspless_mode_supported) {
+			dev_info(dev, "Switching to DSPless mode\n");
+			sdev->dspless_mode_selected = true;
+		} else {
+			dev_info(dev, "DSPless mode is not supported by the platform\n");
```

Comment:
can this be downgraded to a dev_dbg?

Reply:
I personally prefer the dev_info, this would tell us right away if a user mucks with the kernel parameters or if the CI setup is incorrect. I would even make this one a dev_err().

Reply:
dev_info() is I think the right level. I would reserve dev_err for real errors... Can be dev_warn() if we want to make more noise...